tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/SilentAuthTests.cs

```diff
@@ -112,22 +112,42 @@ public async Task SilentAuth_TokenCacheRemainsPersistent_Async()
             }
         }
 
+        [TestMethod]
+        public void Foo()
+        {
+            var pca = PublicClientApplicationBuilder.Create("872cd9fa-d31f-45e0-9eab-6e460a02d1f1").Build();
+
+            pca.UserTokenCache.SetAfterAccess(OnAfterAccess);
+
+            var result = pca.AcquireTokenInteractive(new[] { "499b84ac-1321-427f-aa17-267ca6975798/.default" })
+                .WithPrompt(Prompt.SelectAccount)
+                .ExecuteAsync().Result;
+
+            string at = result.AccessToken; 
```

Comment:
>string at = result.AccessToken;  [](start = 12, length = 32)

what is to be asserted?

Reply:
Reverting, I shouldn't have included this file in the PR.